client/src/components/Navbar.js

```diff
@@ -0,0 +1,33 @@
+import { Outlet, Link } from 'react-router-dom';
+import './Navbar.css'
+
+
+export default function Navbar(){
+  return (
+    <>
+      <nav className="navbar nav-columns">
+        <div className='nav-col-left one-third'>
+          <Link to="/mensproducts">Shop Men</Link>
+          <Link to="/womensproducts">Shop Women</Link>
+        </div>
+        <div className='nav-col-center one-third'>
+          <Link to="/">
+            <p>
+              Pinocelli
+            </p>
+            <p>
+              Studios
+            </p>
+            <p className='premium-cycling-wear'>
+              PREMIUM CYCLING WEAR
+            </p>
+          </Link>
+        </div>
+        <div className='nav-col-right one-third'>
+          <Link to="shoppingcart">Cart</Link>
```

Comment:
I think this needs to be `/shoppingcart`, with a leading `/`, though in this case it probably works without. Well, they all would probably work without. Pick one and be consistent.